test-fixtures/update.py

```diff
@@ -21,10 +21,8 @@
 ]
 '''
 pattern = 'osx-xcode-'
-version_name = 'ventura'
-patterns = [pattern, version_name]
+patterns = [pattern]
 BITRISE_YML = 'bitrise.yml'
-WORKFLOW = 'NewXcodeVersions'
```

Comment:
`NewXcodeVersions` workflow no longer holds the stack version info. The `meta` section, which specifies the stack, is near the bottom instead.

Reply:
you can and for this case you need to set a stack in that workflow so that when there is a new stack version, it is changed only in one workflow, we don't want to change all without trying first